doc/project-plan/public-release-plan.rst

```diff
@@ -0,0 +1,69 @@
+=================================
+Public Release Communication plan
+=================================
+
+Before March 17, 2021, the Aquarium project was developed in a public space
+but had not been announced on a public platform. This document details the
+process the Aquarist Labs organisation will undertake to ensure a smooth
+transition to the public space.
+
+.. list-table::
+   :widths: 10 10 30 15 15
+   :header-rows: 1
+
+   * - Communication Type
+     - Where
+     - Description
+     - Format
+     - Owner
+   * - Tweets
+     - Twitter
+     - Official tweets from the following channels:
+         - Ceph
+         - SUSE
+     - Announcement of Aquarium
+     - All
+   * - Tweets
+     - Twitter
+     - Members of Aquarist Labs to use (if they choose) their personal Twitter
+       account to announce what they have been working on. See below for
+       some text examples.
+     - Announcement of Aquarium
+     - All
+   * - Vlog
```

Comment:
This is new, and previously undiscussed - but I think it's a great way to release a video based on our experiences. Summarise what we've been doing, and then blast this too. Need to add a date option.